docs/platform/what-is-tizen/overview.md

```diff
@@ -30,22 +30,25 @@ From Tizen 3.0, all profiles are built on top of a common, shared infrastructure
 
 ![Common infrastructure for various profiles](media/about_tizen_2.png)
 
-## IoT Capability
+## Multi device experience
```

Comment:
@semun-lee, previously, I suggested sentence-style capitalization for the second heading "Multi device experience" since it was under the modified part. However, it turns out that all the other second headings on this page follow title case. To maintain consistency with other second headings now, I am requesting you to revert back the change. Since it's an urgent PR, we have decided to handle the sentence case issues later on. 

```suggestion
## Multi Device Experience
```